src/layout/css/style.scss

```diff
@@ -69,72 +94,86 @@ span.error, span.failed, span.xpassed, .error .col-result, .failed .col-result,
  *------------------*/
 
 #results-table {
-	border: 1px solid #e6e6e6;
+	border: $border-width solid #e6e6e6;
 	color: #999;
-	font-size: 12px;
-	width: 100%
-}
+	font-size: $font-size-text;
+	width: 100%;
 
-#results-table th, #results-table td {
-	padding: 5px;
-	border: 1px solid #E6E6E6;
-	text-align: left
-}
-#results-table th {
-	font-weight: bold
+	th,
+	td {
+		padding: $spacing;
+		border: $border-width solid #E6E6E6;
+		text-align: left;
+	}
+
+	th {
+		font-weight: bold;
+	}
 }
 
 /*------------------
  * 2. Extra
  *------------------*/
 
-.log:only-child {
-	height: inherit
-}
+$extra-spacing: $spacing;
```

Comment:
Just curious, why do we need to define `$extra-spacing`? Can't we just use `$spacing`?

Reply:
sure can use `$spacing`  and remove `$extra-spacing`.
used `$extra-spacing` i case the original author had other plans for the spacing in different sections.
now, from a readers perspective i would say it is easier to stay with `$spacing`.

any opinions from others?

Reply:
Remove it please if it's not used @jkowalleck 🙏

Reply:
done that.